### PR TITLE
new workflow template for jfrog artifact upload

### PR DESCRIPTION
**Pull request by tristan-takeda**

### CODE REVIEW

File: workflow-templates/jfrog-openshift-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  build-and-push:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3    
+
+      #Sample 'Build' Step
```
Comment (jgleonard-takeda) [inline workflow-templates/jfrog-openshift-deploy.yml:48]: Maybe include some direction towards what a build command may look like

File: workflow-templates/jfrog-openshift-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# This is a starter workflow to help you get started with building Docker Containers using Actions
```
Comment (jgleonard-takeda) [inline workflow-templates/jfrog-openshift-deploy.yml:1]: This comment is from the Docker starter workflow

File: workflow-templates/jfrog-deploy.properties.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (jgleonard-takeda) [inline workflow-templates/jfrog-deploy.properties.json:1]: The metadata file needs to had a filename that matches the workflow file

File: workflow-templates/jfrog-openshift-deploy.yml (hunk 2 above)
Comment (jgleonard-takeda) [reply workflow-templates/jfrog-openshift-deploy.yml:1]: The filename for this workflow also mentions openshift, which isn't in scope at all.

**Review by jgleonard-takeda — CHANGES_REQUESTED** [review]
The filenames need to match, so X.yml and X.properties.json and shouldn't mention openshift.  See the comments in the PR for other asks.

File: workflow-templates/jfrog-openshift-deploy.yml (hunk 1 above)
Comment (tristan-takeda) [reply workflow-templates/jfrog-openshift-deploy.yml]: @jgleonard-takeda I believe I have resolved this, please have a look and let me know if you had something else in mind. Thank you!

**Review by jgleonard-takeda — APPROVED** [review]

**Pull request merged**